src/client/components/MapHeader.tsx

```diff
@@ -307,6 +311,23 @@ const MapHeader = ({
           </Select>
         </Flex>
       </Box>
+      <Box sx={{ position: "relative", mr: "5px" }}>
+        <Button
+          sx={{ ...style.selectionButton }}
+          // eslint-disable-next-line
+          onClick={() => store.dispatch(toggleFind(!findMenuOpen))}
```

Comment:
This is a cleaner way to fix the lint error:
```suggestion
          onClick={() => {
            store.dispatch(toggleFind(!findMenuOpen));
          }
```